index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Twitter
+        </title>
+        <script src="https://kit.fontawesome.com/34321f5008.js" crossorigin="anonymous"></script>
+        <link href="https://fonts.googleapis.com/css2?family=Oxygen:wght@700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="style.css" />
+        <script src="script.js" ></script>
+
+    </head>
+    <body>
+        <div class="content">
+            <div class="colm1">
+                <div class="icons">
+                    <a class="twitter-icon" href="#">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </div>
+            </div>
+            <div class="colm2">
+                <div class ="navbar borderStyle">Home</div>
```

Comment:
If you didn't use 'navbar' class for special property why adding two classes not one?

Reply:
In addition to why using div tag to add text? isn't <p> a good choice for you?

Reply:
I used both of them

Reply:
yeah I think also it would be better I use an h3 tag 
Thanks !!